.github/workflows/tests.yaml

```
@@ -72,3 +72,24 @@ jobs:
 
             -   name: "Run tests"
                 run: "vendor/bin/phpunit --group e2e"
+
+    static-analysis:
+        runs-on: "ubuntu-latest"
+        name: "PHPStan"
```

I would keep the matrix version just because it makes it easier to parametrize the PHP version and reuse it in both the name and install

```suggestion
        name: "PHPStan on PHP ${{ matrix.php }}"
        strategy:
            fail-fast: false
            matrix:
                php:
                    - "8.1"
```